### PR TITLE
Feature/gh 633

### PR DESCRIPTION
**Pull request by developpeurweb**
fixes GH-633

### CODE REVIEW

File: projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.scss [hunk 1]
```diff
@@ -1,132 +1,40 @@
...
+    margin: var(--cx-margin, 0 0 0 0.625rem);
+    height: var(--cx-height, 1.125rem);
+    width: var(--cx-width, 1.125rem);
   }
-  &__tooltip-icon {
-    margin-left: $cx-payment-form-tooltip-icon-margin-left;
-    height: $cx-payment-form-tooltip-icon-size;
-    width: $cx-payment-form-tooltip-icon-size;
+  .cx-payment-form-billing {
+    margin: var(--cx-margin, 0 0 0 1.25rem);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.scss]: This should be bottom not right.

Comment (developpeurweb) [reply projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.scss]: done!

File: projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.html [hunk 2]
```diff
@@ -157,7 +157,7 @@
         </div>
 
         <!-- BUTTON SECTION -->
-        <div class="cx-payment-form__btns row">
+        <div class="row">
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.html]: Shouldn't this be `cx-btns`? In mobile the two buttons stack without any margin 😱

Comment (LTiger14) [reply projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.html]: Or is that part of the follow up

Comment (developpeurweb) [reply projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.html]: I'll take note of this and fix these kind of issues in ticket 1241

Comment (LTiger14) [reply projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.html]: 👍

**Pull request merged**